packages/pharos/src/components/button/pharos-button.ts

```diff
@@ -269,6 +293,10 @@ export class PharosButton extends ScopedRegistryMixin(FocusMixin(AnchorElement))
   }
 
   protected override render(): TemplateResult {
+    // Remove in future release once sufficient time elapsed to update naming convention
+    const a11yLabel = this.a11yLabel ?? this.label;
+    const a11yPressed = this.a11yPressed ?? this.pressed;
```

Comment:
Because this will be released in a major version, I don't believe we need to support backward compatibility like this.

Reply:
My thought was it would encourage faster adoption, it's likely there will be a lot of breaking changes of existing code and the overhead to update everything would require a significant lift.